lib/tropic/tropic_gui.flow

```diff
@@ -58,16 +58,30 @@ export {
 	TGrid(c : [[Tropic]]) -> Tropic;
 
 	// Arabic(RTL) versions
+	TColsRTL(t : [Tropic], isRTL : bool) -> Tropic;
```

Comment:
We don't have M*RTL functions, just M*A, maybe T*RTL should be internal only? AFAIU, the are not intended  to be used outside tropic's implementation

Reply:
T*RTL are used for Material implementation, so they should be exported here. However, I can add a comment about T*A preferability.

Reply:
All this renaming is bad, hence assumes these structs/functions work only with RTL, whilst they implemented to work for both LTR and RTL cases. Must be turned back.